sdk/confidentialledger/Azure.Security.CodeTransparency/src/CreateEntryOperation.cs

```diff
@@ -1,92 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System.Net;
-using System.Threading;
-using System.Threading.Tasks;
-using Azure.Core;
-
-namespace Azure.Security.CodeTransparency
-{
-    /// <summary>
-    /// Tracks the status of a call to <see cref="CodeTransparencyClient.CreateEntry"/>
-    /// or <see cref="CodeTransparencyClient.CreateEntryAsync"/> until completion.
-    /// </summary>
-    internal class CreateEntryOperation : Operation<GetOperationResult>, IOperation
```

Comment:
I think we want to resuscitate it to be able for the client to wait for submission completion, i.e. `submission.WaitForCompletion();`

Reply:
Restored and updated based on the latest API changes